src/services/GraphDataSource.ts

```diff
@@ -216,10 +234,21 @@ export default class GraphDataSource {
 
   // Some helpers
 
-  public fetchForApp = (duration: DurationInSeconds, namespace: string, app: string, injectServiceNodes?: boolean) => {
+  public fetchForApp = (
+    duration: DurationInSeconds,
+    namespace: string,
+    app: string,
+    includeHealth?: boolean,
+    injectServiceNodes?: boolean
+  ) => {
```

Comment:
Quite minor, but I would like to know if we can omit these changes.

I wouldn't like to see the an increased number of parameters on these helpers. At some point, I think it will be better to invoke the generic `fetchGraphData` which lets you fully customize the request.

Reply:
I'll look at this, perhaps the helpers are no longer helpful.